Resources/Prototypes/Nyanotrasen/Catalog/Fills/Items/belt.yml

```diff
@@ -0,0 +1,9 @@
+- type: entity
+  id: ClothingBeltKatanaSheathFilled
+  parent: ClothingBeltKatanaSheath
+  suffix: Filled
+  components:
+  - type: StorageFill
+    contents:
+      - id: Katana
```

Comment:
Is this supposed to have 2 different swords in it or should these be an orGroup?

Reply:
It is supposed to have both a katana and a wakizashi.